client/src/components/LeftSidebar/components/Categorical/Categorical.tsx

```diff
@@ -294,6 +295,52 @@ class Categorical extends React.Component<Props, CategoricalState> {
                 />
               ))}
           </>
+        ) : isVcpDeployment ? (
+          <>
+            {/* DATASET FIELDS - Combined standard and author categories when VCP deployment is enabled */}
+            {(standardCategoryNames.length > 0 ||
+              authorCategoryNames.length > 0) && (
+              <Collapse>
+                <span>Dataset Categories</span>
+                <>
+                  {standardCategoryNames.map((catName: string) => (
+                    <Category
```

Comment:
nit: this pattern is repeated a lot and you could replace with something like:
```typescript
    const mkCategoryMaker = function (catType: string): (catName: string) => ReactNode {
      return function (catName: string): ReactNode {
        return <Category
                 key={catName}
                 metadataField={catName}
                 onExpansionChange={this.onExpansionChange}
                 isExpanded={expandedCategories.includes(catName)}
                 categoryType={catType}
        />;
      };
    }
... 
     {standardCategoryNames.length && standardCategoryNames.map(mkCategoryMaker("standard"))}
```
 but maybe that's too abstract (writing a function to generate a function to pass to `map`)